src/go/api/strelka/strelka.proto

```diff
@@ -5,7 +5,9 @@ service Frontend {
   rpc ScanFile(stream ScanFileRequest) returns (stream ScanResponse) {}
   rpc CompileYara(stream CompileYaraRequest) returns (stream CompileYaraResponse) {}
   rpc SyncYara(stream SyncYaraRequest) returns (stream SyncYaraResponse) {}
+  rpc SyncYara_v2(stream SyncYaraRequest_v2) returns (stream SyncYaraResponse) {}
```

Comment:
Leaving v1 for now for backward compatibility. We can remove once Mantis has been updated with the new RPC call and has been widely deployed.